platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/version/WorkflowVersionRunnableValidator.java

```diff
@@ -0,0 +1,138 @@
+package com.softicar.platform.workflow.module.workflow.version;
+
+import com.softicar.platform.common.code.reference.point.SourceCodeReferencePoints;
+import com.softicar.platform.common.core.i18n.IDisplayString;
+import com.softicar.platform.core.module.uuid.AGUuid;
+import com.softicar.platform.emf.permission.CurrentEmfPermissionRegistry;
+import com.softicar.platform.emf.validation.result.EmfDiagnosticCollection;
+import com.softicar.platform.workflow.module.WorkflowI18n;
+import com.softicar.platform.workflow.module.workflow.node.action.AGWorkflowNodeAction;
+import com.softicar.platform.workflow.module.workflow.node.action.permission.AGWorkflowNodeActionPermission;
+import com.softicar.platform.workflow.module.workflow.node.precondition.AGWorkflowNodePrecondition;
+import com.softicar.platform.workflow.module.workflow.transition.AGWorkflowTransition;
+import com.softicar.platform.workflow.module.workflow.transition.permission.AGWorkflowTransitionPermission;
+import java.util.Optional;
+import java.util.function.Function;
+
+public class WorkflowVersionRunnableValidator {
```

Comment:
To me, this class name carries a strong implication of `java.lang.Runnable`.
I think we should either rename this to `WorkflowVersionConsistencyValidator` or so, **or** at least provide a definition (in Javadoc) of what "runnable" is supposed to mean, in the context of a workflow (since that is definitely not self-explanatory).